core/src/datayoga_core/blocks/map/block.py

```diff
@@ -1,37 +1,52 @@
 import json
 import logging
 from abc import ABCMeta
-from typing import Any, Dict, List, Optional
+from typing import Any, Dict, List, Optional, Union
 
 from datayoga_core import expression
 from datayoga_core.block import Block as DyBlock
 from datayoga_core.context import Context
+from datayoga_core.expression import Language
 from datayoga_core.result import BlockResult, Result, Status
 
 logger = logging.getLogger("dy")
 
 
 class Block(DyBlock, metaclass=ABCMeta):
 
+    @staticmethod
+    def _prepare_expression(language: Language, expr: Union[dict, str]) -> str:
```

Comment:
please add a clear comment explaining this block and its purpose

Reply:
✅